macros/macros.yml

```diff
@@ -8,5 +8,10 @@ macros:
         description: Source schema name
       - name: destination_schema
         description: Destination schema name
+      - name: source_database
+        description: Source database name (optional)
```

Comment:
I would add here docs stating that the target database is used if not value is provided.

Reply:
great point!